File: v2.x/Examples/SciChartDemo/SciChartDemo/Utils/RandomUtil.m
//
//  RandomUtil.m
//  SciChartDemo
//
//  Created by Admin on 19/04/2017.
//  Copyright © 2017 ABT. All rights reserved.
//

#import "RandomUtil.h"
#define ARC4RANDOM_MAX 0x100000000

@implementation RandomUtil

+(double)nextDouble {
    return ((double)arc4random() / ARC4RANDOM_MAX);
}

@end

File: v2.x/Examples/SciChartSwiftDemo/SciChartSwiftDemo/Utils/RandomUtil.swift
//
//  RandomUtil.swift
//  SciChartSwiftDemo
//
//  Created by Admin on 19/04/2017.
//  Copyright © 2017 SciChart Ltd. All rights reserved.
//

import UIKit

class RandomUtil: NSObject {
    static let ARC4RANDOM_MAX:Double = 0x100000000

    static func nextDouble() -> Double {
        return (Double(arc4random()) / ARC4RANDOM_MAX)
    }
    
}
